src/interfaces/IController.ts

```diff
@@ -20,6 +20,7 @@ export interface GetTvlRequest {
   chain: string;
   block: string;
   date: string;
+  autointegrationParams: AutointegrationParams;
```

Comment:
Did you test it, I think we need to split AutointegrationParams into GetTvlRequest